include/zephyr/net/wifi_credentials.h

```diff
@@ -56,6 +62,18 @@ struct wifi_credentials_header {
 	uint32_t timeout;                 /**< Timeout for connecting to the network */
 	uint8_t bssid[WIFI_MAC_ADDR_LEN]; /**< BSSID (Basic Service Set Identifier) */
 	uint8_t channel;                  /**< Channel on which the network operates */
+	/**
+	 * < Anonymous identifier
```

Comment:
are you using `<` as a special identifier?

Reply:
The `<` character does not work here, it tells doxygen that the variable we are commenting is on left side of the comment which is not the case here.